src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+# Positron Emission Tomography
+
+Support for Positron Emission Tomography (PET) was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+The following example PET datasets have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+-   One subject single dynamic scan (pet, mri, blood):
+    [`pet_example1`](https://www.dropbox.com/sh/1wpmmilq18mjquq/AABmA1o4vZMmBuGK1Io3vwf3a?dl=0)
+
+-   One subject single dynamic scan (pet, mri):
+    [`pet_example2`](https://www.dropbox.com/sh/33adqfaq3kkn58z/AADF5ZbMp6YOs2lOZgnLxZpaa?dl=0)
+
+-   One subject single dynamic scan (pet, blood):
+    [`pet_example3`](https://www.dropbox.com/sh/6egwsy442caribr/AABw6GrfvBQClbgie5Gaa6WLa?dl=0)
+
+Further PET datasets are available from [OpenNeuro](https://openneuro.org).
+
+## Terminology and conventions
+
+PET-BIDS is fully consistent with the BIDS specification as a whole.
+However, BIDS was initially developed in the context of MRI,
+so some terminology may be unfamiliar to researchers from each field.
+This section adds clarifications to
+[Common Principles - Definitions](../02-common-principles.md#definitions)
+for the PET context, and introduces the term "time zero" which is currently
+specific to PET.
+
+1.  **Session** - In most cases, a new session with respect to PET corresponds
```

Comment:
From Mattia Veronese: Would it be possible to give some examples of different types of session as done for fMRI in https://bids-specification.readthedocs.io/en/bep-009/06-longitudinal-and-multi-site-studies.html  to show what to do in case of break, multiple tracer at baseline?

Reply:
We are in the process of making more examples and will share them under the bids-examples.

Reply:
The additonal examples have been shared as of now on Dropbox and will be added to the above after they are moved to the bids-examples repo.